src/components/Navbar.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from 'react-router-dom';
+
+const links = [
+  { path: '/', text: 'Books' },
+  { path: 'categories', text: 'Categories' },
+];
+function NavBar() {
+  return (
+    <nav style={{ display: 'flex', alignItems: 'center', gap: '40px' }}>
+      <h1 style={{ fontFamily: 'monospace', fontSize: '30px' }}>Mamun Book Store</h1>
+      <ul style={{
+        display: 'flex',
+        listStyleType: 'none',
+        gap: '80px',
+      }}
```

Comment:
- [ ] [OPTIONAL] Please strongly consider putting these styles in an external stylesheet and using class names instead as it is not ideal to have the styles laid out in your JSX like this 👍

Reply:
@codepantha inshallah i will